cmd/root.go

```diff
@@ -933,25 +937,45 @@ func CreateWebhook(webhook Webhook) {
 				webhook.Repository,
 			),
 		)
-		// set up the encoding reader from the current webhook
-		webhookToUpdate := WebHookPatch{
-			URL:          webhook.Config.URL,
-			Content_Type: webhook.Config.Content_Type,
-			Insecure_SSL: webhook.Config.Insecure_SSL,
-			Secret:       webhook.Config.Secret,
-		}
-		reader := GetReaderFromObject(webhookToUpdate)
+		// instantiate a Webhook struct for the response
 		webhookResponse := Webhook{}
+		// update the status and events
+		webhookToUpdate := WebHookUpdate{
+			Active: webhook.Active,
+			Events: webhook.Events,
+		}
+		readerUpdate := GetReaderFromObject(webhookToUpdate)
 		writeErr = dstRestClient.Patch(
 			fmt.Sprintf(
-				"repos/%s/%s/hooks/%d/config",
+				"repos/%s/%s/hooks/%d",
 				dstOrganization,
 				webhook.Repository,
 				webhook.ID,
 			),
-			reader,
+			readerUpdate,
```

Comment:
Will this capture new data `webHookUpdate` in additional to other data? I guess that is `webhookResponse`

Reply:
So, there are two API endpoints being called. The first updates `Events` and `Status`. If that fails for some reason (ID being incorrect, it will pass to the error state and report. If it succeeds, it moves on to the Config patching endpoint where it is allowed to update the secret, URL, and other data points.

`webhookResponse` just holds the response data, which only gets populated on HTTP 200 codes. I don't really need it at all. I could just do an `_`.

The `writeErr` is what holds any erros that occur.

Reply:
Make sure to test it, and I am good. I will also test to make sure